app/src/main/res/values-ru/strings.xml

```diff
@@ -17,33 +17,33 @@
     <string name="lblScanExternal">Сканировать внешнее хранилище</string>
     <string name="lblNotificationMalwareDetectionTitle">Обнаружение вредоносного ПО</string>
     <string name="lblNotificationMalwareDetectionDescription">Используется для оповещения при обнаружении вредоносного ПО</string>
-    <string name="lblNotificationRealtimeTitle">Сканер в реальном времени</string>
+    <string name="lblNotificationRealtimeTitle">Онлайн-сканер</string>
```

Comment:
Real-time vs online?

Reply:
The line seemed too long to me and it took me a long time to figure out how to shorten the translation. Now I'm more immersed in online and real-time scanners, and I agree that it makes sense to keep the old translation.

Reply:
Online works, I just don't want people to get the impression that files might be sent off.